bertinNDAYISHIMIYE/capstoneProject-MyBrand
# Ft swagger doc

###What does this PR do?
>add blog routes documentation
>add enquiries route documentation
>add users route documentation

###Description of the task to be completed!
>Document the api using with swagger

###How should this be tested manually?
>clone this repo with https://github.com/bertinNDAYISHIMIYE/capstoneProject-MyBrand
>checkout to ft-swaggerDoc branch
>start the server with `npm run dev`
>check the browser using the url of documentation `http://localhost:3030/api-docs/`


###Trello card
https://trello.com/c/P2qO0LSJ

###Screenshot
<img width="1399" alt="Screen Shot 2020-12-21 at 18 01 28" src="https://user-images.githubusercontent.com/72761713/102796271-af5abb80-43b6-11eb-89bc-bc5a8ac88c62.png">
